tests/app/test_cloudfoundry_config.py

```diff
@@ -16,7 +16,9 @@ def notify_config():
             'admin_client_secret': 'admin client secret',
             'secret_key': 'secret key',
             'dangerous_salt': 'dangerous salt',
-            'allow_ip_inbound_sms': ['111.111.111.111', '100.100.100.100']
+            'allow_ip_inbound_sms': ['111.111.111.111', '100.100.100.100'],
+            'route_secret_key_1': "key_1",
```

Comment:
Should you test four configs?

- key 1 only
- key 2 only
- key 1 and key 2
- no keys

Reply:
Case 1-3 are addressed in test_authentication.py. Will build logic around case 4